content/en/docs/kubernetes/operator/troubleshooting/target-allocator.md

```diff
@@ -0,0 +1,488 @@
+Kubernetes cluster.
+
+### Do you know if metrics are actually being scraped?
+
+After you’ve deployed all of your resources to Kubernetes, make sure that the
+Target Allocator is discovering scrape targets from your
+[`ServiceMonitor`](https://prometheus-operator.dev/docs/operator/design/#servicemonitor)(s)
+or
+[`PodMonitor`](https://prometheus-operator.dev/docs/user-guides/getting-started/#using-podmonitors)(s).
+
+Suppose that you have this `ServiceMonitor` definition:
+
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: sm-example
+  namespace: opentelemetry
+  labels:
+    app.kubernetes.io/name: py-prometheus-app
+    release: prometheus
+spec:
+  selector:
+    matchLabels:
+      app: my-app
+  namespaceSelector:
+    matchNames:
+      - opentelemetry
+  endpoints:
+    - port: prom
+      path: /metrics
+    - port: py-client-port
+      interval: 15s
+    - port: py-server-port
+```
+
+this `Service` definition:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: py-prometheus-app
+  namespace: opentelemetry
+  labels:
+    app: my-app
+    app.kubernetes.io/name: py-prometheus-app
+spec:
+  selector:
+    app: my-app
+    app.kubernetes.io/name: py-prometheus-app
+  ports:
+    - name: prom
+      port: 8080
+```
+
+and this `OpenTelemetryCollector` definition:
+
+```yaml
+apiVersion: opentelemetry.io/v1beta1
+kind: OpenTelemetryCollector
+metadata:
+  name: otelcol
+  namespace: opentelemetry
+spec:
+  mode: statefulset
+  targetAllocator:
+    enabled: true
+    serviceAccount: opentelemetry-targetallocator-sa
+    prometheusCR:
+      enabled: true
+      podMonitorSelector: {}
+      serviceMonitorSelector: {}
+  config:
+    receivers:
+      otlp:
+        protocols:
+          grpc: {}
+          http: {}
+      prometheus:
+        config:
+          scrape_configs:
+            - job_name: 'otel-collector'
+              scrape_interval: 10s
+              static_configs:
+                - targets: ['0.0.0.0:8888']
+
+    processors:
+      batch: {}
+
+    exporters:
+      logging:
+        verbosity: detailed
+
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [logging]
+        metrics:
+          receivers: [otlp, prometheus]
+          processors: []
+          exporters: [logging]
+        logs:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [logging]
+```
+
+First, set up a `port-forward` in Kubernetes, so that you can expose the Target
+Allocator service:
+
+```shell
+kubectl port-forward svc/otelcol-targetallocator -n opentelemetry 8080:80
+```
+
+Where `otelcol-targetallocator` is the value of `metadata.name` in your
+`OpenTelemetryCollector` CR concatenated with the `-targetallocator` suffix, and
+`opentelemetry` is the namespace to which the `OpenTelemetryCollector` CR is
+deployed.
```

Comment:
we could also give people the query for the service by doing `kubectl get svc -l app.kubernetes.io/component=opentelemetry-targetallocator`

Reply:
Thanks! Added that.